cdk/src/open-data-platform/data-plane/data-import/write-water-systems-data-handler.handler.ts

```diff
@@ -116,31 +187,29 @@ export async function handler(_: APIGatewayProxyEvent): Promise<APIGatewayProxyR
 
   const s3Params = {
     Bucket: 'opendataplatformapistaticdata',
-    Key: 'pwsid_lead_connections.geojson',
+    Key: 'pwsid_lead_connections_even_smaller.geojson',
   };
 
-  let db: ConnectionPool | undefined;
-
-  // Read CSV file and write to water systems table.
   try {
-    db = await connectToDb();
-    if (db == undefined) {
-      throw Error('Unable to connect to db');
-    }
+    const rdsService = new AWS.RDSDataService();
```

Comment:
can you link rds data api documentation in a comment here?

Reply:
Done